### PR TITLE
GIMP 3.0 RC2

### PR DESCRIPTION
**Pull request by brunvonlope**
I do not intend, for now, to do the release myself, I am just anticipating part of the job

### CODE REVIEW

File: org.gimp.GIMP.json [hunk 1]
```diff
@@ -152,8 +152,8 @@
             "sources" : [
                 {
                     "type": "archive",
-                    "url": "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.3.1.tar.gz",
-                    "sha256": "58aad2b32c047070a52f1205b309bdae007442e0f983120e4ff57551eb6f10f1",
+                    "url": "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.3.2.tar.gz",
+                    "sha256": "20e40baa6654785d69642e6e85542968db3c5d08da630adc590ff066a52c5938f4ce8a77c0097e00010a905c8c31d8f131eb0308a3f8b6439ab6be4133eae246",
```
Comment (hfiguiere) [inline org.gimp.GIMP.json:156]: this is not a sha256. it's way too long.

Comment (brunvonlope) [reply org.gimp.GIMP.json:156]: Clunky copy paste error of mine

**Comment by Jehan** [comment]
@brunvonlope 

> I do not intend, for now, to do the release myself, I am just anticipating part of the job

You are welcome to do the package release. I am not holding on to this task as a precious property. On the contrary, I've been trying to delegate the Flatpak maintenance to others for years (and you're doing a wonderful job at it!). 💌

**Comment by brunvonlope** [comment]
@Jehan Yeah, I'm not afraid or anything like that. I try do build things as much as possible to relieve your (and others) shoulders so they are light to code but the flatpak release I needed to clarify that I will not be able to tackle yet. Next year for sure.

**Comment by brunvonlope** [comment]
"Image Graph" was NOT tested since it isn't displayed on stable releases.

PDF import/export, EXR import/export working as expected.

AVIF and HEIC are working too just updating x265 (which was broken: https://github.com/flathub/org.gimp.GIMP/pull/356#issuecomment-2351081639). Now, I pushed the libheif update to see if AVIF/HEIC still works.

**Comment by brunvonlope** [comment]
bot, build

**Comment by brunvonlope** [comment]
bot, build

**Comment by Jehan** [comment]
@brunvonlope Is it normal that your last commit is titled "Revert COMMIT_SHA"? Did you intend to actually paste here an actual commit hash?

**Comment by brunvonlope** [comment]
> Did you intend to actually paste here an actual commit hash?

Yes. It is fff51c9e471291df9f116cd939f83a7dd8cddd2d

**Comment by Jehan** [comment]
Thanks @brunvonlope. I updated your commit message locally so it will have the info when I'll force push next.

**Pull request merged**